mptt/managers.py

```diff
@@ -616,6 +616,7 @@ def move_node(self, node, target, position='last-child'):
         move the node yourself by setting node.parent.
         """
         self._move_node(node, target, position=position)
+        node.save()
```

Comment:
Yes, also it's more logical, as `parent` is really a model field, and not part of the MPTT optimisation.